doc/BLOG.md

```diff
@@ -0,0 +1,140 @@
+
+Configure both repositories:
+```
+[global]
+log-level-console=info
+start-fast=y
+
+repo1-path=/var/lib/pgbackrest/1
+repo1-retention-full=2
+
+repo2-path=/var/lib/pgbackrest/2
+repo2-retention-full=2
+repo2-bundle=y
+repo2-block=y
+
+[demo]
+pg1-path=/var/lib/postgresql/12/demo
+```
+
+Create the stanza on both repositories:
+```
+$ pgbackrest --stanza=demo stanza-create
+```
+
+The block incremental backup feature is best demonstrated with a larger dataset. In particular, we would prefer to have at least one
+file that is near the maximum segment size of 1GB. This can be accomplished by creating data with `pgbench`:
+```
+$ /usr/lib/postgresql/12/bin/pgbench -i -s 65
+```
+
+## File Bundling
+
+File bundling stores data in the repository more efficiently by combining smaller files together. This results in fewer files overall in the backup which improves the speed of all repository operations, especially on object stores like S3, Azure, and GCS. There may also be cost savings on repositories that have a cost per operation since there will be fewer lists, deletes, etc.
+
+To demonstrate this we'll make a backup on repo1, which does not have bundling enabled:
+```
+$ pgbackrest --stanza=demo --type=full --repo=1 backup
+```
+
+Now we check the number of files in repo1 for the latest backup:
+```
+$ find /var/lib/pgbackrest/1/backup/demo/latest/ -type f | wc -l
+
+991
+```
+
+This is pretty normal for a small database without bundling enabled since each file is stored separately. There are also a few metadata files that pgBackRest uses to track the backup.
+
+Now we'll perform the same actions on repo2, which has file bundling enabled:
+```
+$ pgbackrest --stanza=demo --type=full --repo=2 backup
+$ find /var/lib/pgbackrest/2/backup/demo/latest/ -type f | wc -l
+
+7
+```
+
+This time there are far fewer files. The small files have been bundled together and zero-length files are stored only in the manifest.
+
+The `repo-bundle-size` option can be used to control the maximum size of bundles before compression and other operations are applied.
+The `repo-bundle-limit` option limits the files that will be added to bundles. It is not a good idea to set these options too large because any failure in the bundle on backup or restore will require the entire bundle to be retried. The goal of file bundling is to combine small files -- there is very seldom any benefit in combining larger files.
+
+## Block Incremental Backup
+
+Block incremental backup saves space in the repository by storing only the parts of the file that have changed since the last backup. The block size depends the file size and when the file was last modified, i.e. larger, older files will get larger block sizes. Blocks are compressed and encrypted into super blocks that can be retrieved independently to make restore more efficient.
```

Comment:
`The block size depends ON the file size...`

Reply:
Fixed in b7ec9f102fb.